commands/bootstrap/setup.go

```diff
@@ -546,6 +551,22 @@ func installAnsible(version string) error {
 	return err
 }
 
+// installParamiko installs Paramiko needed when openSSH is not used
+// Install using pip3 if available else using pip
+func installParamiko() error {
+	pipCmd, err := getPipCmd()
+	if err != nil {
+		return err
+	}
+
+	// Installing ansible
+	cmd := exec.Command(pipCmd, "install", "--upgrade", "paramiko")
```

Comment:
As per [comment on Forge PR](https://github.com/ystia/forge/pull/137#discussion_r501508861) should we use a specific or bounded version for paramiko?

Reply:
As the Ansible doc doesn't specify any specific or bounded version, proposing to let it like this

Reply:
ok